source/server/http_test_server_filter_config.cc

```diff
@@ -31,7 +31,7 @@ class HttpTestServerDecoderFilterConfig
     return Envoy::ProtobufTypes::MessagePtr{new nighthawk::server::ResponseOptions()};
   }
 
-  std::string name() override { return "test-server"; }
+  std::string name() const override { return "test-server"; }
```

Comment:
Similar question here, not sure I follow if this is related to updating Envoy. Do we want a separate PR with description that would explain these additional changes?

Alternatively, please help me understand why they are necessary here.

Reply:
> Can we hold off this PR for about a day? Eric is trying to import Nighthawk into our internal codebase. We are hoping to import just Nighthawk, not all of Envoy and this PR might break / delay that effort. We should be done with the import by tomorrow end of day.

Sure no problem